### PR TITLE
Add Korean-friendly font

### PR DESCRIPTION
**Pull request by corydalis10**
Changed font to Pretendard for better Korean readability!
Please review my PR.

### CODE REVIEW

File: css/ko.css [hunk 1]
```diff
@@ -4,6 +4,10 @@
 */
 /* For image callouts in writing-middleware.md */
 
+@import url('../fonts/ko/Pretendard.css');
+
+html, body {font-family: Pretendard;}
```
Comment (bjohansebas) [inline css/ko.css]: It's advisable to add at least one alternative font besides Pretendard, although I believe it wouldn't be necessary in this case as there's already an alternative in the main styles. Am I correct?

Comment (ghost) [reply css/ko.css]: @bjohansebas Correct. I'll add Arial, sans-serif font for alternative.

**Comment by bjohansebas** [comment]
LGTM

**Review by bjohansebas — APPROVED** [review]

**Review by UlisesGascon — APPROVED** [review]

**Comment by bjohansebas** [comment]
Thanks @rivercory

**Pull request merged**